docs/docs/Tutorials/mcp-tutorial.md

```diff
@@ -0,0 +1,238 @@
+* [An OpenAI API key](https://platform.openai.com/api-keys)
+
+This tutorial uses an OpenAI LLM. If you want to use a different provider, you need a valid credential for that provider.
+
+## Create an agentic flow
+
+1. In Langflow, click **New Flow**, and then select the [**Simple agent**](/simple-agent) template.
+2. In the **Agent** component, enter your OpenAI API key.
+    If you want to use a different provider or model, edit the **Model Provider**, **Model Name**, and **API Key** fields accordingly.
+3. To test the flow, click <Icon name="Play" aria-hidden="true" /> **Playground**, and then ask the LLM `Is it safe to go hiking in the Adirondacks today?`
+
+    The LLM response is vague, though the Agent does know the current date by using its internal `get_current_date` function.
+
+    ```
+    Today is July 11, 2025.
+    To determine if it's safe to go hiking in the Adirondacks today, you should check the current weather conditions, trail advisories, and any local alerts (such as bear activity or flooding).
+    Would you like a detailed weather forecast or information on trail conditions for the Adirondacks today?
+    ```
+
+    For improved, up-to-date context for your Agent, replace the connected tools with a connection to a weather MCP server.
+
+## Add an MCP Tools component
+
+    :::tip Manage MCP Servers
+    You can manage your MCP servers' configurations in the **Settings** menu, but you still need an **MCP Tools** component in your flow for each individual server.
+    :::
+
+1. Remove the **URL** and **Calculator** tools, and then drag the [**MCP Tools**](/mcp-client) component into your workspace.
+2. In the **MCP Tools** component, click <Icon name="Plus" aria-hidden="true"/> **Add MCP Server**.
+
+3. There are multiple ways to install MCP servers, which are covered in the [**MCP Tools**](/mcp-client) component page.
+
+    This example installs an [MCP weather server](https://github.com/isdaniel/mcp_weather_server) to your local machine with uv and Python.
+
+    Make sure you install the server in the same Python environment where Langflow is running.
+    * If you are running Langflow in a virtual environment, activate that environment before installing the server.
+    * If you are using Docker, install the package inside the Docker container.
+    * If you are running Langflow system-wide, install the package globally or in the same user environment.
+
+    To install the server, run the following command:
+    ```shell
+    uv pip install mcp_weather_server
+    ```
+
+4. Configure the MCP server.
+
+    In the **Add MCP Server** pane, select either **JSON** or **STDIO** to enter the configuration for starting your server.
+    Both options configure an MCP server in Langflow with a command and arguments, which Langflow executes to launch the server process when needed.
+    Both are included here to demonstrate how the STDIO commands can be filled in from the JSON configuration values.
```

Comment:
Maybe don't **need** both, but I think it's important, because there are a lot of different ways people and companies present their MCP servers in repos.